### DISCUSSION TITLE
Some useful icons are not available there

### DISCUSSION THREAD

**Discussion by dev-rahulprasad** [tailwindlabs/heroicons#1227] [Ideas]
1. Social media icons
2. Disabled

I will add more suggestions if I get result

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose?

For "disabled" perhaps consider:
![image](https://github.com/user-attachments/assets/38b5ef00-a1c6-431e-a5db-5dbc01082de7)

**Comment by dev-rahulprasad** [comment]
Does it support tailwind css?

**Reply by zefir-git** [reply]
Doesn't need to, it's just SVG. You can add any Tailwind CSS classes you like to the SVGs. If you want to change the icon colour using `text-color-…` make sure to set `fill="currentColor"` or use TailwindCSS [`fill-…`](https://tailwindcss.com/docs/fill)
